<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Выберите настройки игры</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>85</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);
font: 11pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Введите свой игровой ник</string>
   </property>
  </widget>
  <widget class="QPushButton" name="launch">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(211, 0, 0);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Начать</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/play.png</normaloff>images/play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="hard">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>хардкор</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="medi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>средний</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lat">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> лайт</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
